lib/AL_USDMaya/AL/usdmaya/Global.cpp

```diff
@@ -98,6 +99,56 @@ AL::event::CallbackId Global::m_preRead;
 AL::event::CallbackId Global::m_postRead;
 AL::event::CallbackId Global::m_fileNew;
 
+//----------------------------------------------------------------------------------------------------------------------
+//class of MObjects
+MSelectionList selected;
```

Comment:
To avoid exporting a DSO symbol, and to make it clear we have a global variable.... 
```c++
static MSelectionList g_selected;
```